schema_based_on_diagram.sql

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE patients (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(100),
+  date_of_birth DATE,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE medical_histories (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  admitted_at TIMESTAMP,
+  patient_id INT,
+  status VARCHAR(100),
+  PRIMARY KEY(id),
+  FOREIGN KEY(patient_id) REFERENCES patients(id)
+);
+
+CREATE TABLE invoices (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  total_amount DECIMAL(5,2),
+  generated_at TIMESTAMP,
+  payed_at TIMESTAMP,
+  medical_history_id INT,
+  PRIMARY KEY(id),
+  FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id)
+);
+
+CREATE TABLE medical_hitories_treatments(
+  treatments_id INT,
+  medical_hitories_id INT,
+  PRIMARY KEY(treatments_id,medical_hitories_id),
+  FOREIGN key (treatments_id) REFERENCES treatments(id),
+  FOREIGN key (medical_histories_id) REFERENCES medical_histories(id)
+);
```

Comment:
- [ ] Great work setting up the tables 👍🏼, I just want to congratulate you and mention that you have a typo here, kindly make sure to fix it so you can move to the next milestone.
```suggestion
CREATE TABLE medical_histories_treatments(
  treatments_id INT,
  medical_histories_id INT,
  PRIMARY KEY(treatments_id,medical_histories_id),
  FOREIGN key (treatments_id) REFERENCES treatments(id),
  FOREIGN key (medical_histories_id) REFERENCES medical_histories(id)
);
```
Keep the Good work 🚀